R/plotNorm.R

```diff
@@ -8,7 +8,7 @@
 #' Normalization is performed by edgeR::calcNormFactors. Note TMM is specifically tailored to count-based
 #' data.  Thus this function is only appropriate for count-based data.
 #'
-#' @param DGEdata  A DGEobj or counts matrix.
+#' @param DGEdata Name of DGEobj with a class of DGEobj.
```

Comment:
I do not think that is an accurate documentation. As user do not pass the object name she/he pass the object itself.

Reply:
done